packages/devtools_app/lib/src/app.dart

```diff
@@ -533,58 +534,48 @@ class _VerboseLoggingSetting extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-      padding: const EdgeInsets.only(
-        right: 6.0,
-        bottom: 6.0,
-      ),
-      decoration: const BoxDecoration(
-        border: Border.fromBorderSide(
-          BorderSide(width: 0.5, color: Colors.grey),
+    return Column(
+      children: [
+        Row(
+          children: [
+            CheckboxSetting(
+              label: const Text('Enable verbose logging'),
+              listenable: preferences.verboseLoggingEnabled,
+              toggle: (enable) => preferences.toggleVerboseLogging(enable),
+              gaItem: gac.verboseLogging,
+            ),
+            const SizedBox(width: defaultSpacing),
+            IconLabelButton(
+              label: 'Copy logs',
+              icon: Icons.copy,
+              onPressed: () async => await copyToClipboard(
+                LogStorage.root.toString(),
+                'Successfully copied logs',
+              ),
+            ),
+            const SizedBox(width: defaultSpacing),
+            ClearButton(
+              label: 'Clear logs',
+              onPressed: LogStorage.root.clear,
+            ),
+          ],
         ),
-      ),
-      child: Column(
-        children: [
-          Row(
+        Container(
```

Comment:
remove the container and use `mainAxisAlignment: MainAxisAlignment.center` on the Row